RoboJackets/urc-software
# Feat/bt testing

# Description
This PR does the following:
(More features will be added as this is a work in progress to validate all parts of the implementation of path planning).
- Creates a new node for publishing a costmap, placed in `urc_test`
- Refactors the `tester` package into `urc_test`, remove unnecessary files that test for linting.
- Deals with a few typos in the urc_bt package.

# Testing Steps (if relevant)
1. `colcon build ; . install/setup.bash`
2. `ros2 launch urc_bringup bringup.launch.py`

Expectation: Should generate a costmap and start the behavior tree

# Design Presentation (only required for major additions)
Post the link to your presentation here and [be sure to update the presentation document](../documents/design/README.md) if this PR results in a significant addition to the stack

# Self Checklist
